trulens_eval/examples/dummy_example.ipynb

```diff
@@ -0,0 +1,126 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dummy Example\n",
+    "\n",
+    "This notebook shows the use of the dummy feedback function provider which\n",
+    "behaves like the huggingface provider except it does not actually perform any\n",
+    "network calls and just produces constant results. It can be used to prototype\n",
+    "feedback function wiring for your apps before invoking potentially slow (to\n",
+    "run/to load) feedback functions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "from pathlib import Path\n",
+    "import sys\n",
+    "\n",
+    "# If running from github repo, can use this:\n",
+    "sys.path.append(str(Path().cwd().parent.resolve()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import threading\n",
+    "\n",
+    "from examples.frameworks.custom.custom_app import CustomApp\n",
+    "\n",
+    "from trulens_eval import Feedback\n",
+    "from trulens_eval import FeedbackMode\n",
+    "from trulens_eval import Tru\n",
+    "from trulens_eval.feedback.provider.hugs import Dummy\n",
+    "from trulens_eval.feedback.provider.hugs import Huggingface\n",
+    "from trulens_eval.tru_custom_app import TruCustomApp\n",
+    "from trulens_eval.utils.threading import TP\n",
+    "\n",
+    "tru = Tru()\n",
+    "\n",
+    "tru.reset_database()\n",
+    "\n",
+    "tru.start_dashboard(\n",
+    "    force = True,\n",
+    "    _dev=Path().cwd().parent.resolve()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# hugs = Huggingface()\n",
+    "hugs = Dummy()\n",
+    "\n",
+    "f_positive_sentiment = Feedback(hugs.positive_sentiment).on_output()"
```

Comment:
How does the dummy have a positive_sentiment attribute?

Reply:
It has the same feedback functions as Huggingface except they always produce the same result, without making any actual huggingface requests.